render-ws-java-client/src/test/java/org/janelia/render/client/BlockLayoutCreatorTest.java

```diff
@@ -134,4 +133,40 @@ public void specifyingDimensionTwiceThrowsError() {
 			assertEquals("Intervals for dimension x already specified", e.getMessage());
 		}
 	}
+
+	@Test
+	public void sameMaxZForRegularAndShiftedGrids() {
+
+		final int blockSizeXY = 9000;
+		final int blockSizeZ = 1;
+		final Bounds bounds = new Bounds(73.0, 164.0, 11986.0,
```

Comment:
Where do the numbers come from? Is this a bug that occurred in practice or just random numbers? In the first case, I would add a short comment. In the second case, I'm wondering if there was a less random set of numbers that show the same properties of the system under test (maybe in a more straightforward way)?

Reply:
This is a bug that occurred in practice.  I added a comment to explain.